lib/publiccloud/gcp_client.pm

```diff
@@ -33,13 +33,9 @@ sub init {
     $self->project_id($data->{project_id});
     $self->account($data->{client_id});
     assert_script_run('source ~/.bashrc');
-    if (is_sle('>=15')) {
-        # kill it in case it's running
-        script_run("killall chronyd && sleep 5 && if pgrep chronyd; then killall -9 chronyd; fi");
-        assert_script_run("chronyd -q 'pool time.google.com iburst'");
-    } else {
-        assert_script_run('ntpdate -s time.google.com');
-    }
+    # kill it in case it's running
+    assert_script_run('systemctl stop chronyd.service && sleep 5 && if pgrep chronyd; then pkill -9 chronyd; fi');
```

Comment:
```suggestion
    assert_script_run('systemctl stop chronyd.service');
```
If you assert running `systemctl stop chronyd.service`, then I think you can assume that it's stopped. I think we can remove the rest.